modules/acquia_cms_tour/src/Form/WelcomeModalForm.php

```diff
@@ -66,7 +66,7 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state, $options = NULL) {
-    $acms_logo = $this->profileExtensionList->getPath('acquia_cms') . '/acquia_cms.png';
+    $logo = file_url_transform_relative(file_create_url(theme_get_setting('logo.url')));
```

Comment:
@panshulK Both of this functions are deprecated: https://www.drupal.org/node/2940031
We should find alternative and use it. 
**PS:** Make sure the functions are available in Drupal 9.3 & 9.2.

Reply:
Done!